.prettierrc

```diff
@@ -0,0 +1,8 @@
+{
+  "singleQuote": true,
+  "trailingComma": "all",
+  "arrowParens": "always",
+  "semi": true,
+  "printWidth": 80,
+}
+ 
```

Comment:
Let's make sure we have a newline at the end of the file :smile:

Reply:
Done

Reply:
Done